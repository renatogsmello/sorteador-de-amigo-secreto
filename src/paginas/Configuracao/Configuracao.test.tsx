import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import Configuracao from ".";

const mockNavegacao = jest.fn();

jest.mock("react-router-dom", () => {
  return {
    useNavigate: () => mockNavegacao,
  };
});

describe("A página de configuração", () => {
  test("Deve ser renderizada corretamente", () => {
    const { container } = render(
      <RecoilRoot>
        <Configuracao />
      </RecoilRoot>
    );

    expect(container).toMatchSnapshot();
  });
});
