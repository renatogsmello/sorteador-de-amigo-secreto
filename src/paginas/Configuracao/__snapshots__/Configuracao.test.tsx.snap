// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A página de configuração Deve ser renderizada corretamente 1`] = `
<div>
  <div
    class="bg-[#FFF9EB] border-2 border-black border-t-[64px] box-border p-20 flex flex-1 -mt-8 justify-center"
  >
    <section>
      <h2>
        Vamos começar!
      </h2>
      <form>
        <div
          class="mb-8"
        >
          <input
            class=" rounded-s-[45px] h-[82px] w-2/3 box-border pl-8 text-xl border-2 border-black shadow-[0_2px_0_1px_rbg(0,0,0)] focus:outline-none"
            placeholder="Insira os nomes dos participantes"
            type="text"
            value=""
          />
          <button
            class="rounded-e-[45px] h-[82px] w-1/3 box-border border-2 border-black text-xl text-black shadow-[2px_2px_0_1px_rbg(0,0,0)] cursor-pointer bg-[#c4c4c4] hover:opacity-80 disabled:opacity-60 disabled:cursor-not-allowed "
            disabled=""
          >
            Adicionar
          </button>
        </div>
        
      </form>
      <ul />
      <footer
        class="flex items-center justify-between md:flex-col"
      >
        <button
          class="w-[350px] height-[80px] text-xl shadow-[2px_2px_0_1px_rgb(0,0,0)] rounded-[45px] bg-[#fe652b] text-white cursor-pointer disabled:opacity-60 disabled:cursor-not-allowed hover:bg-[#4b69fd] py-8"
          disabled=""
        >
          Iniciar a brincadeira
        </button>
        <img
          alt="Sacolas de compras"
          class="md:mt-8"
          src="/imagens/sacolas.png"
        />
      </footer>
    </section>
  </div>
</div>
`;
